rca/enquire_to_study/forms.py

```diff
@@ -57,11 +60,10 @@ class EnquireToStudyForm(forms.Form):
     def __init__(self, *args, **kwargs):
         super(EnquireToStudyForm, self).__init__(*args, **kwargs)
         # Placeholder
-        self.fields["first_name"].widget.attrs["placeholder"] = "First name *"
-        self.fields["last_name"].widget.attrs["placeholder"] = "Last name *"
-        self.fields["email"].widget.attrs["placeholder"] = "Email *"
-        self.fields["phone_number"].widget.attrs["placeholder"] = "Phone number *"
-        self.fields["city"].widget.attrs["placeholder"] = "City or town of residence *"
+
+        # Set initial values
+        self.fields["country_of_residence"].initial = ("GB", "United Kingdon")
```

Comment:
This can be done sitewide and declarative:

Show certain countries first
Provide a list of country codes as the COUNTRIES_FIRST setting and they will be shown first in the countries list (in the order specified) before all the alphanumerically sorted countries.

If you want to sort these initial countries too, set the COUNTRIES_FIRST_SORT setting to True.

By default, these initial countries are not repeated again in the alphanumerically sorted list. If you would like them to be repeated, set the COUNTRIES_FIRST_REPEAT setting to True.

Finally, you can optionally separate these ‘first’ countries with an empty choice by providing the choice label as the COUNTRIES_FIRST_BREAK setting.

https://pypi.org/project/django-countries/#customize-the-country-list

Reply:
Thanks @BrianXu20, This isn't regarding the order of the values, it's so the field loads with the initial value already selected:

![image](https://user-images.githubusercontent.com/2128707/113720491-99af9e80-96e6-11eb-9ab4-7fa8641a4da1.png)